php/libraries/Candidate.class.inc

```diff
@@ -613,8 +613,31 @@ class Candidate
 
         return $subprojList;
     }
+
+    /**
+     * Returns the SubprojectID and subproject title of the candidate's last visit
+     * based on Date_visit
+     *
+     * @return array|null Array with values SubprojectID and title
+     *                    for the candidate's last visit or NULL if none exist
+     */
+    public function getLastVisitSubproject(): ?array
```

Comment:
This is a bit of a nit-pick but can you rename this to something like `getSubprojectForMostRecentVisit()`? I find the current name a bit unclear.

Reply:
I think a lengthy function name is the lesser of two evils compared with an ambiguous one